pkg/config/defaults.go

```diff
@@ -87,6 +87,9 @@ func setDefaults(local bool) {
 
 	viper.SetDefault("security.audit_check_interval", 24*time.Hour)
 	viper.SetDefault("security.audit_check_url", "https://audit.lakefs.io/audit")
+	viper.SetDefault("security.check_latest_version", true)
+	viper.SetDefault("security.check_latest_version_cache", 24*time.Hour)
```

Comment:
If you rename the above, should rename here as well

Reply:
https://github.com/treeverse/lakeFS/pull/5608#discussion_r1155895308